apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -144,12 +123,35 @@ end_per_testcase(_Case, Config) ->
 %% ============================================================
 
 start_first_node(Config) ->
-    aecore_suite_utils:start_node(dev1, Config),
-    aecore_suite_utils:connect(aecore_suite_utils:node_name(dev1)),
+    [ Dev1 | _ ] = proplists:get_value(devs, Config),
+    aecore_suite_utils:start_node(Dev1, Config),
+    aecore_suite_utils:connect(aecore_suite_utils:node_name(Dev1)),
     ok.
 
-test_subscription(_Config) ->
-    N = aecore_suite_utils:node_name(dev1),
+start_second_node(Config) ->
+    [ Dev1, Dev2 | _ ] = proplists:get_value(devs, Config),
+    N1 = aecore_suite_utils:node_name(Dev1),
+    N2 = aecore_suite_utils:node_name(Dev2),
+    aecore_suite_utils:start_node(Dev2, Config),
+    aecore_suite_utils:connect(N2),
+    timer:sleep(2000),
```

Comment:
Shouldn't this be changed to `aecore_suite_utils:await_aehttp(N2)`?

Reply:
Yes.

Reply:
I see two approvals. Merging this.
I am preparing separate PR for optimizing this.

Reply:
doh merge conflict

Reply:
Testing merge on my fork https://travis-ci.org/lucafavatella/epoch/builds/320305550
Then will attempt to push as GitHub recommends (force-pushing would require new approvals).

Reply:
The protected branch hook does not allow me to push to master as GitHub recommends.

Reply:
Opened PR #541.